components/devices.css/assets/scss/devices.scss

```diff
@@ -0,0 +1,448 @@
+@import 'mixins';
+
+.marvel-device{
+	display: inline-block;
+	position: relative;
+	-webkit-box-sizing: content-box;
+	-moz-box-sizing: content-box;
+	box-sizing: content-box;
+
+	.screen {
+		width: 100%;
+		position: relative;
+		height: 100%;
+		color: white;
+		z-index: 2;
+		text-align: center;
+		display: block;
+		@include border-radius(1px);
+		-webkit-box-shadow: 0 0 0 3px #111;
+		box-shadow: 0 0 0 3px #111;
+	}
+
+	.top-bar, .bottom-bar {
+		height: 3px;
+		background: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.